WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsViewModel.swift

```diff
@@ -221,6 +226,21 @@ private extension SubscriptionsViewModel {
         return duration
     }
 
+    /// Returns the site plan expiration date formatted as "MMMM d". e.g: "August 11",
+    /// or nil if expired or we can't retrieve the expiration date
+    ///
+    func formattedExpirationDate(for plan: WPComSitePlan) -> String? {
+        guard daysLeft(for: plan) >= 0 else {
+            return nil
+        }
+        guard let planExpiryDate = plan.expiryDate else {
+            return nil
+        }
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "MMMM d"
```

Comment:
For better localisation, we should use `dateFormatter.setLocalizedDateFormatFromTemplate("MMMM d")` here. Using `.dateFormat` will always render in the style `August 11`, which is a US-centric approach, and we shouldn't assume that locale, because it will stand out in a bad way when (in future) other locale users see it.

[See this page for a helpful example](https://developer.apple.com/documentation/foundation/dateformatter) (in Apple's docs, of all places?! I was astonished...)